app/page.tsx

```
@@ -7,13 +7,13 @@ const LandingPage = () => {
       <header className="flex justify-between items-center p-4 bg-white">
         <div className="text-2xl font-bold text-[#6c63ff]"></div>
         <nav className="space-x-4">
-          <a href="#" className="text-black hover:underline">
+          <a href="/" className="text-black hover:underline">
```

ESLint

[error] Do not use an `<a>` element to navigate to `/`. Use `<Link />` from `next/link` instead. See: https://nextjs.org/docs/messages/no-html-link-for-pages

(@next/next/no-html-link-for-pages)